### ISSUE TITLE
Windows 2016 LTSC version with .net 4.7.1

### ISSUE THREAD

**Issue by jordyvaneijk** [Microsoft/mssql-docker#395]
Is there a version (tag) that is LTSC and has .net framework 4.7.1 install on it? I cant seem to find it.

**Comment by Pirlo-Zhang** [comment]
@jordyvaneijk 

Just use the latest tag, which will be updated along with Windows Server 2016, the current LTSC product.

See:

[Choosing Container OS versions
](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility#choosing-container-os-versions)

**Comment by jordyvaneijk** [comment]
If i use this tag i get the following error

`Step 1/12 : FROM microsoft/mssql-server-windows-express:4.7.1-runtime-20181113-windowsservercore-ltsc2016
manifest for microsoft/mssql-server-windows-express:4.7.1-runtime-20181113-windowsservercore-ltsc2016 not found`

This tag is nog available for MSSQL docker image.. It is for dotnet framework. But this issue is specifically about MSSQL on windowsservercore ltsc

**Comment by Pirlo-Zhang** [comment]
@jordyvaneijk 

Yes, the tag is for .net framework,  there's no available similar tag for SQL Server docker windows image.

Consider to build custom windows container and do a test.

[Create a base image](https://docs.docker.com/develop/develop-images/baseimages/)
